tests/OnlineSales.Tests/SimpleTableTests.cs

```diff
@@ -48,17 +50,39 @@ public virtual async Task UpdateItemNotFoundTest()
     }
 
     [Fact]
-    public async Task CreateAndUpdateItemTest()
+    public virtual async Task CreateAndCheckEntityState_ChangeLog()
     {
         var testCreateItem = await CreateItem();
 
-        var testUpdateItem = UpdateItem(testCreateItem.Item1);
+        var item = await GetTest<T>(testCreateItem.Item2);
 
-        await PatchTest(testCreateItem.Item2, testUpdateItem!);
+        var result = Get<ChangeLog>(c => c.ObjectId == item!.Id && c.ObjectType == typeof(T).Name && c.EntityState == EntityState.Added);
 
-        var item = await GetTest<T>(testCreateItem.Item2);
+        result.Should().NotBeNull();
+    }
 
-        item.Should().BeEquivalentTo(testCreateItem.Item1);
+    [Fact]
+    public async Task CreateAndUpdateItemTest()
```

Comment:
How is it related with ChangeLog testing?

Reply:
I think here your concern is Why Did I Change the CreateAndUpdateItemTest? I made a single method as "CreateAndUpdateItem" and referred to it in both places (Change long one and CreateAndUpdateItemTest)